### ISSUE TITLE
AC and IR Device not shown as Devices

### ISSUE THREAD

**Issue by ridhoperdana** [prasad-edlabadka/homebridge-tuya-ir#1]
Hi, thank you for creating this awesome project.

I've a question (or maybe a bug report). Why my IR and AC is not shown as devices at homebridge menu or even at the Home app on iOS?

I've add the config like below:
![Screen Shot 2021-06-16 at 05 42 32](https://user-images.githubusercontent.com/13913371/122132840-cc53e100-ce65-11eb-8b70-500048474fc8.png)

Thank you

**Comment by prasad-edlabadka** [comment]
IR blaster will not be shown as device. AC should show up. Are you devices in “America” region? Can you please add your debug log? I.e. Run homebridge in debug mode and send the logs?

**Comment by ridhoperdana** [comment]
yes, it is on America region. When I run the homebridge with debug, i can't identify any error from homebridge-tuya-ir since the log is joined with other plugin.
Here is my AC Devices that already registered at Tuya, is it because the type is `Other` ? @prasad-edlabadka 

![Screen Shot 2021-06-16 at 18 04 00](https://user-images.githubusercontent.com/13913371/122208534-9f3b1900-cecd-11eb-910c-116f2fbd1190.png)

Thanks!

**Comment by prasad-edlabadka** [comment]
It's difficult to debug without logs. If you are logging home bridge logs to file, can you please grep [TuyaIR] and send the logs?

**Comment by ridhoperdana** [comment]
Hi, here is the log from TuyaIR
```
[6/17/2021, 8:14:26 AM] Registering platform 'homebridge-tuya-ir.TuyaIR'
[6/17/2021, 8:14:26 AM] [TuyaIR] Initializing TuyaIR platform...
[6/17/2021, 8:14:26 AM] [TuyaIR] Finished initializing platform: TuyaIR
[6/17/2021, 8:14:26 AM] [TuyaIR] Executed didFinishLaunching callback
[6/17/2021, 8:14:26 AM] [TuyaIR] Starting discovery...
[6/17/2021, 8:14:26 AM] [TuyaIR] Trying to login...
[6/17/2021, 8:14:26 AM] [TuyaIR] Logging in to the the server https://openapi.tuyaus.com...
[6/17/2021, 8:14:27 AM] [TuyaIR] API call successful.
[6/17/2021, 8:14:27 AM] [TuyaIR] Logged in. Fetching configured remotes...
[6/17/2021, 8:14:27 AM] [TuyaIR] Calling endpoint https://openapi.tuyaus.com/v1.0/infrareds/04068050e8db84c76ed4/remotes
[6/17/2021, 8:14:27 AM] [TuyaIR] API call successful.
```

thank you @prasad-edlabadka

**Comment by jyhur805** [comment]
Hi, thank you for tuya-ir plugin.
I've the same problem too.
Why my AC or IR blaster is not shown as devices at homebridge accessorie menu.  (•̩̩̩̩＿•̩̩̩̩)

------------------------ here is the log from my homebridge ----------------------------------
[6/17/2021, 7:07:15 AM] Homebridge v1.3.4 (Homebridge B080) is running on port 53572.
Initializing HAP-NodeJS v0.9.4...
[6/17/2021, 7:07:15 AM] [TuyaIR] Launched child bridge with PID 13583
[6/17/2021, 7:07:16 AM] Registering platform 'homebridge-tuya-ir.TuyaIR'
[6/17/2021, 7:07:16 AM] [TuyaIR] Loaded homebridge-tuya-ir v0.0.2 child bridge successfully
[6/17/2021, 7:07:16 AM] [TuyaIR] Finished initializing platform: TuyaIR
Preparing Advertiser for 'TuyaIR 320C' using bonjour-hap backend!
[6/17/2021, 7:07:16 AM] [TuyaIR] Executed didFinishLaunching callback
[6/17/2021, 7:07:16 AM] [TuyaIR] Starting discovery...
[6/17/2021, 7:07:16 AM] [TuyaIR] Trying to login...
[6/17/2021, 7:07:16 AM] [TuyaIR] Logging in to the the server https://openapi.tuyaus.com...
Starting to advertise 'TuyaIR 320C' using bonjour-hap backend!
[6/17/2021, 7:07:16 AM] Homebridge v1.3.4 (TuyaIR) is running on port 35896.
[6/17/2021, 7:07:16 AM] [TuyaIR] API call successful.
[6/17/2021, 7:07:16 AM] [TuyaIR] Logged in. Fetching configured remotes...
[6/17/2021, 7:07:16 AM] [TuyaIR] Calling endpoint https://openapi.tuyaus.com/v1.0/infrareds/250538472cf43287193c/remotes
[6/17/2021, 7:07:17 AM] [TuyaIR] API call successful.
[6/17/2021, 9:07:11 AM] [TuyaIR] Need to refresh token now...
[6/17/2021, 9:07:12 AM] [TuyaIR] API call successful.
==================================================================
thank you @prasad-edlabadka

**Comment by jyhur805** [comment]
Hi, thank you for tuya-ir plugin.
I've the same problem too.
Why my AC or IR blaster is not shown as devices at homebridge accessorie menu.  (•̩̩̩̩＿•̩̩̩̩)

```
------------------------ here is the log from my homebridge ----------------------------------
[6/17/2021, 7:07:15 AM] Homebridge v1.3.4 (Homebridge B080) is running on port 53572.
Initializing HAP-NodeJS v0.9.4...
[6/17/2021, 7:07:15 AM] [TuyaIR] Launched child bridge with PID 13583
[6/17/2021, 7:07:16 AM] Registering platform 'homebridge-tuya-ir.TuyaIR'
[6/17/2021, 7:07:16 AM] [TuyaIR] Loaded homebridge-tuya-ir v0.0.2 child bridge successfully
[6/17/2021, 7:07:16 AM] [TuyaIR] Finished initializing platform: TuyaIR
Preparing Advertiser for 'TuyaIR 320C' using bonjour-hap backend!
[6/17/2021, 7:07:16 AM] [TuyaIR] Executed didFinishLaunching callback
[6/17/2021, 7:07:16 AM] [TuyaIR] Starting discovery...
[6/17/2021, 7:07:16 AM] [TuyaIR] Trying to login...
[6/17/2021, 7:07:16 AM] [TuyaIR] Logging in to the the server https://openapi.tuyaus.com...
Starting to advertise 'TuyaIR 320C' using bonjour-hap backend!
[6/17/2021, 7:07:16 AM] Homebridge v1.3.4 (TuyaIR) is running on port 35896.
[6/17/2021, 7:07:16 AM] [TuyaIR] API call successful.
[6/17/2021, 7:07:16 AM] [TuyaIR] Logged in. Fetching configured remotes...
[6/17/2021, 7:07:16 AM] [TuyaIR] Calling endpoint https://openapi.tuyaus.com/v1.0/infrareds/250538472cf43287193c/remotes
[6/17/2021, 7:07:17 AM] [TuyaIR] API call successful.
[6/17/2021, 9:07:11 AM] [TuyaIR] Need to refresh token now...
[6/17/2021, 9:07:12 AM] [TuyaIR] API call successful.
==================================================================
```
thank you @prasad-edlabadka

**Comment by prasad-edlabadka** [comment]
Closed by mistake

**Comment by prasad-edlabadka** [comment]
Fixed in v0.0.6
